src/site/site.service.ts

```diff
@@ -65,6 +69,11 @@ export class SiteService {
             throw new Error(`Site initialisation failed for ${host}`);
         }
 
+        const internalAccount = await this.accountService.createInternalAccount(
+            newSite,
+            'index',
```

Comment:
Use `ACTOR_DEFAULT_HANDLE` from consts maybe?

Reply:
Will do this with a cleanup of site service to use consts for tablenames too